src/AspNetCoreModuleV2/InProcessRequestHandler/dllmain.cpp

```diff
@@ -119,10 +120,18 @@ CreateApplication(
         if (!FAILED_LOG(hr = IN_PROCESS_APPLICATION::Start(*pServer, *pHttpApplication, pParameters, nParameters, inProcessApplication)))
         {
             *ppApplication = inProcessApplication.release();
+            if (ANCMEvents::ANCM_START_APPLICATION_SUCCESS::IsEnabled(pServer->GetTraceContext()))
```

Comment:
Would it make sense to have a single helper method like `RaiseEventIfEnabled()`?